cmd/minishift/cmd/start.go

```diff
@@ -409,26 +409,49 @@ func exportContainerImages(hostVm *host.Host, version string) {
 	fmt.Println(fmt.Sprintf("-- Exporting of OpenShift images is occuring in background process with pid %d.", exportCmd.Process.Pid))
 }
 
-// calculateDiskSizeInMB converts a human specified disk size like "1000MB" or "1GB" and converts it into Megabits
-func calculateDiskSizeInMB(humanReadableDiskSize string) int {
-	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
+func calculateMemorySize(memorySize string) int {
+	if stringUtils.HasOnlyNumbers(memorySize) {
+		memorySize += "M"
```

Comment:
So something is not quite kosher here. In the case of 'calculateMemorySize' you use as unit in case if it is not specified 'M'. Then in the end you divide by units.MiB

In calculateDiskSize you use the same unit 'M' in case nothing is specified, but then you divide by units.MB.